lms/djangoapps/support/static/support/jsx/entitlements/components/Main/Main.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import SearchContainer from '../Search/SearchContainer';
+
+const Main = () => {
+  return (
+    <div>
+      <h1>
```

Comment:
a11y concern: Do we really want this to be `<h1>`?  There should only even be one `<h1>` on a page. For us this would be the Header for edx.  You should be starting at the level according to the parent header like the outline of a paper.

If this is still a placeholder that is fine, but we should avoid making it H1.